api/task_processor/thread_monitoring.py

```diff
@@ -1,34 +1,23 @@
 import json
 import logging
-import os
-import typing
-from threading import Thread
 
-UNHEALTHY_THREADS_FILE_PATH = "/tmp/task-processor-unhealthy-threads.json"
+from pydantic import BaseModel
 
-logger = logging.getLogger(__name__)
-
-
-def clear_unhealthy_threads():
-    if _unhealthy_threads_file_exists():
-        os.remove(UNHEALTHY_THREADS_FILE_PATH)
+THREAD_COUNTS_FILE_PATH = "/tmp/task-processor-thread-counts.json"
```

Comment:
It's not really clear why this file exists. A docstring or comment somewhere would probably be helpful.

Reply:
Yep, good point, I'll add.